core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -473,7 +474,7 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     val rawCwd = System.getProperty("user.dir")
     val cwd = if (Utils.isWindows) s"/$rawCwd".replace("\\", "/") else rawCwd
     assertResolves("hdfs:/root/spark.jar", "hdfs:/root/spark.jar")
-    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:/root/spark.jar#app.jar")
+    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:///root/spark.jar#app.jar")
```

Comment:
yes let me make it clear. the function `assertResolves(before: String, after: String)` have two params, it means we need to check the before value whether equals the after value which we want when we call `Utils.resolveURI(uri)`, right? otherwise the param `before: String` make no sense.
e.g  the after value of  `Utils.resolveURI("hdfs:///root/spark.jar#app.jar").toString` should be "hdfs:///root/spark.jar#app.jar" rather than "hdfs:/root/spark.jar#app.jar"